src/schema/datatypes/pet.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# First group
+- suffixes:
+    - pet
+  extensions:
+    - .nii.gz
+    - .nii
+    - .json
+  entities:
+    sub: required
+    ses: optional
+    task: optional
+    acq: optional
+    rec: optional
+    run: optional
+# Second group
+- suffixes:
+    - blood
+  extensions:
+    - .tsv
+    - .json
+  entities:
+    sub: required
+    ses: optional
+    task: optional
+    acq: optional
+    rec: optional
+    run: optional
+    recording: optional
```

Comment:
```suggestion
    recording: required
```

Reply:
This realtes to the comment https://github.com/bids-standard/bids-specification/pull/633#pullrequestreview-501369897